force-app/main/default/classes/UTIL_OrgTelemetry_TEST.cls

```diff
@@ -124,6 +124,21 @@ private class UTIL_OrgTelemetry_TEST {
             UTIL_OrgTelemetry.TelemetryParameterName.IsEnabled_StoreErrors.name(),
             edaSettings.Store_Errors_On__c
         );
+
+        assertBooleanValue(
```

Comment:
Just an opinion: Not sure why we have a long method to verify all feature parameters. We usually break  long methods into smaller one (one test per concept), but all other feature param checks have been added prior so perhaps it is fine for this case?

Reply:
I'd vote to keep it as is, given we will have to reprocess the metric calculations for each feature parameter (impact upon test execution times).